content/en/docs/concepts/cluster-administration/kubelet-garbage-collection.md

```diff
@@ -44,7 +44,7 @@ Default is 85%.
 2. `image-gc-low-threshold`, the percent of disk usage to which image garbage collection attempts
 to free. Default is 80%.
 
-We also allow users to customize garbage collection policy through the following kubelet flags:
+Users can customize garbage collection policy through the following kubelet flags:
```

Comment:
This would also work:
```suggestion
You can customize the garbage collection policy through the following kubelet flags:
```

Reply:
@sftim that sounds good, but I was intentionally imitating the above sentence on line 40:
```
Users can adjust the following thresholds ...
```

Reply:
I'll certainly add `the` though...I didn't notice that one.

Reply:
applying my own suggestion :)
```suggestion
Users can customize the garbage collection policy through the following kubelet flags:
```

Reply:
@sftim I can also use `You` in the first sentence.

What do you think?

Reply:
> also use You in the first sentence.

I'd prefer that fix @bl-ue.